lib/queuePopulator/IngestionPopulator.js

```diff
@@ -590,14 +590,28 @@ class IngestionPopulator {
     }
 
     _setupUpdatedReaders(done) {
+        const readyReaders = [];
```

Comment:
small change: rather than using this extra var, just push into `this.logReaders`